lighthouse-core/config/config.js

```diff
@@ -106,29 +106,22 @@ function cleanTrace(trace) {
   return trace;
 }
 
-function filterPasses(passes, audits, rootPath) {
+function validatePasses(passes, audits, rootPath) {
+  if (!Array.isArray(passes)) {
```

Comment:
I forgot this isn't returning anything, just logging now, so that's fine, actually